### PR TITLE
Ru ru translation

### PR DESCRIPTION
**Pull request by alex-0xFF**
Translating ru-RU

### CODE REVIEW

File: data/language/ru-RU.txt [hunk 1]
```diff
@@ -564,8 +564,8 @@ STR_0559    :Большое тематическое здание со пуга
 STR_0560    :Место, где гости, которым стало плохо, могут прийти в себя
 STR_0561    :Цирковое представление с животными под куполом большого тента
 STR_0562    :Самоходные вагончики едут по мульти-уровневому пути, проезжая страшные декорации и специальные эффекты.
-STR_0563    :Сидя в комфортных вагончиках, только с простыми поручнями, пассажиры наслаждаются большими мягкими выпадами и извилистым путём, так же часто, как и {ENDQUOTES}временем в воздухе{ENDQUOTES} на возвышенностях.
-STR_0564    :Бегущий по деревянному пути, этот аттракцион быстр, жёсткий, шумный и дающие ощущение неуправляемой поездки с большим количеством {ENDQUOTES}времени в воздухе{ENDQUOTES}.
+STR_0563    :Сидя в комфортных вагончиках, только с простыми поручнями, пассажиры наслаждаются большими мягкими выпадами и извилистым путём, так же часто, как и 'временем в воздухе' на возвышенностях.
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Why did you replace these? As far as I know, `'` is not a proper Russian quotation mark, with Russian mostly using „this style“ (which is supported in OpenRCT2).

File: data/language/ru-RU.txt [hunk 2]
```diff
@@ -835,13 +835,14 @@ STR_0830    :{SMALLFONT}{BLACK}Приблизить
...
 STR_0833    :{SMALLFONT}{BLACK}Пауза
-STR_0834    :{SMALLFONT}{BLACK}Настройки 
+STR_0834    :{SMALLFONT}{BLACK}Настройки
 STR_0835    :Ошибка во время инициализации игры
 STR_0836    :Невозможно запустить игру в минимизированном состоянии
 STR_0837    :Не удалось инициализировать графическую систему
 STR_0838    :<removed string - do not use>
 STR_0839    :{UINT16} x {UINT16}
 STR_0840    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{UINT16} x {UINT16}
+# The following six strings were used for display resolutions, but have been replaced.
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: This remark will be removed from English soon, so you can just remove it.

File: data/language/ru-RU.txt [hunk 3]
```diff
@@ -1164,16 +1165,16 @@ STR_1159    :{SMALLFONT}{BLACK}Разместить декорации, сады
...
+STR_1164    :{STRINGID}{NEWLINE}(Пр. щелчок - удалить)
 STR_1165    :{STRINGID} - {STRINGID} {COMMA16}
 STR_1166    :Нельзя опустить здесь уровень воды...
 STR_1167    :Нельзя поднять здесь уровень воды...
 STR_1168    :Опции
 STR_1169    :(Нет)
 STR_1170    :{STRING}
-STR_1171    :{RED}Закрыто - - 
-STR_1172    :{YELLOW}{STRINGID} - - 
+STR_1171    :{RED}Закрыто - -
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: These two lines _require_ a trailing whitespace.

File: data/language/ru-RU.txt [hunk 4]
```diff
@@ -1344,10 +1345,10 @@ STR_1339    :{BLACK}Нет результатов теста...
 STR_1340    :{WINDOW_COLOUR_2}Макс. скорость: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Длительн.: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: This one too.

File: data/language/ru-RU.txt [hunk 5]
```diff
@@ -1344,10 +1345,10 @@ STR_1339    :{BLACK}Нет результатов теста...
 STR_1340    :{WINDOW_COLOUR_2}Макс. скорость: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Длительн.: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} / 
+STR_1343    :{DURATION} /
 STR_1344    :{WINDOW_COLOUR_2}Длина: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1345    :{LENGTH}
-STR_1346    :{LENGTH} /
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: And this one too.

File: data/language/ru-RU.txt [hunk 6]
```diff
@@ -1731,8 +1732,8 @@ STR_1726    :Земля не продаётся!
 STR_1727    :Права на строительство не продаются!
 STR_1728    :Здесь нельзя купить права на строительство...
 STR_1729    :Земля не принадлежит парку!
-STR_1730    :{RED}Закрыто - -
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: These lines also need the trailing whitespace

File: data/language/ru-RU.txt [hunk 7]
```diff
@@ -1779,7 +1780,7 @@ STR_1774    :Допускается только один кабельный л
 STR_1775    :Выкл.
 STR_1776    :Вкл.
 STR_1777    :{WINDOW_COLOUR_2}Музыка:
-STR_1778    :{STRINGID} - - 
+STR_1778    :{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: As does this one.

File: data/language/ru-RU.txt [hunk 8]
```diff
@@ -1887,12 +1888,14 @@ STR_1882    :Каждые 45 минут
...
 STR_1884    :Каждые 2 часа
 STR_1885    :Никогда
-STR_1886    :Наблюдение {STRINGID}
+STR_1886    :Осмотр {STRINGID}
 STR_1887    :{WINDOW_COLOUR_2}Время с послед. проверки: {BLACK}{COMMA16} мин.
 STR_1888    :{WINDOW_COLOUR_2}Время послед. инспекции: {BLACK}более 4 часов
 STR_1889    :{WINDOW_COLOUR_2}Простой: {MOVE_X}{255}{BLACK}{COMMA16}%
 STR_1890    :{SMALLFONT}{BLACK}Выберите, как часто механики проверяют аттракцион
 STR_1891    :{STRINGID} ещё нет в парке!
+# The following two strings were used to display an error when the disc was missing.
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: You can get rid of this remark.

File: data/language/ru-RU.txt [hunk 9]
```diff
@@ -2787,8 +2780,8 @@ STR_2779    :Вид #{COMMA16}
 STR_2780    :Дополнительный вид
 # End of new strings
 STR_2781    :{STRINGID}:{MOVE_X}{255}{STRINGID}
-STR_2782    :SHIFT + 
-STR_2783    :CTRL + 
+STR_2782    :SHIFT +
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: These lines also need trailing whitespace.

File: data/language/ru-RU.txt [hunk 10]
```diff
@@ -2983,10 +2976,10 @@ STR_2974    :Альт. схема 3
 STR_2975    :{SMALLFONT}{BLACK}Выберите, какую схему изменить или применить
 STR_2976    :{SMALLFONT}{BLACK}Раскрасить отдельную часть аттракциона, используя выбранную схему.
 STR_2977    :Имя сотрудника
-STR_2978    :Введите новое имя этого сотрудника: 
+STR_2978    :Введите новое имя этого сотрудника:
 STR_2979    :Невозможно назвать сотрудника...
 STR_2980    :Слишком много баннеров
-STR_2981    :{RED}Нет входа - - 
+STR_2981    :{RED}Нет входа - -
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Needs trailing whitespace.

File: data/language/ru-RU.txt [hunk 11]
```diff
@@ -3168,16 +3161,16 @@ STR_3159    :список
 STR_3160    :OpenRCT2: Запускается первый раз...
 STR_3161    :<removed string - do not use>
 STR_3162    :Невозможно выделить память
-STR_3163    :Установка данных: 
+STR_3163    :Установка данных:
 STR_3164    :{BLACK}{COMMA16} выбрано (макс. {COMMA16})
 STR_3165    :
-STR_3166    :{BLACK}(ID: 
+STR_3166    :{BLACK}(ID:
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Also needs trailing whitespace.

File: data/language/ru-RU.txt [hunk 12]
```diff
@@ -3475,7 +3546,7 @@ STR_5342    :<removed string - do not use>
 STR_5375    :{UP}
 STR_5376    :{DOWN}
 STR_5462    :{CURRENCY}
-STR_5467    :ALT + 
+STR_5467    :ALT +
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Needs trailing whitespace.

**Comment by Gymnasiast** [comment]
Overall this looks pretty good, but several strings in our language files need to have a trailing whitespace. You removed all trailing whitespace, even the required ones. Those will need to be restored.

**Comment by alex-0xFF** [comment]
First question. 
Can I add remarks into ru-RU, which are present in en-GB, till they will be removed? I made some setup for translation, which relies on strings order.
It's OK, if not, I just have to do add some code to script.

File: data/language/ru-RU.txt (hunk 1 above)
Comment (AaronVanGeffen) [reply data/language/ru-RU.txt]: Try `{LEFTGUILLEMET}` and `{RIGHTGUILLEMET}`.

Comment (Gymnasiast) [reply data/language/ru-RU.txt]: ^

File: data/language/ru-RU.txt [hunk 13]
```diff
@@ -1732,7 +1732,7 @@ STR_1727    :Права на строительство не продаются!
 STR_1728    :Здесь нельзя купить права на строительство...
 STR_1729    :Земля не принадлежит парку!
 STR_1730    :{RED}Закрыто - - 
-STR_1731    :{WHITE}{STRINGID} - - 
+STR_1731    :{WHITE}{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Needs a trailing space.

File: data/language/ru-RU.txt [hunk 14]
```diff
@@ -3319,17 +3309,17 @@ STR_3310    :{WINDOW_COLOUR_2}{LENGTH}
...
+STR_3316    :Введите описание этого сценария:
 STR_3317    :Нет деталей
 STR_3318    :{SMALLFONT}{BLACK}Выберите, в какой группе появится этот сценарий
 STR_3319    :{WINDOW_COLOUR_2}Группа:
 STR_3320    :Невозможно сохранить сценарий...
 STR_3321    :Новые объекты установлены
 STR_3322    :{WINDOW_COLOUR_2}Задача: {BLACK}{STRINGID}
-STR_3323    :Отсутствую данные объекта, ID: 
-STR_3324    :Требует Add-On: 
+STR_3323    :Отсутствую данные объекта, ID:
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Needs a trailing space. The next one too.

File: data/language/ru-RU.txt [hunk 15]
```diff
@@ -3356,7 +3346,7 @@ STR_3347    :Аттракцион слишком большой, слишком
 STR_3348    :Переимен.
 STR_3349    :Удалить
 STR_3350    :Имя дизайна трека
-STR_3351    :Введите новое имя для этого дизайна: 
+STR_3351    :Введите новое имя для этого дизайна:
```
Comment (Gymnasiast) [inline data/language/ru-RU.txt]: Also needs a trailing space.

**Pull request merged**